README.md

```diff
@@ -1,586 +1,15 @@
-   1. Under General Information:
-      - Enter the **Algorithm Name** as `gedi-subsetter`.
-      - Enter the **Algorithm Description** as `Subset GEDI L1B, L2A, L2B, or L4A granules within an area of interest (AOI)`.
-      - Enter the **Disk Space** as `20GB`.
-      - Enter the **Resource Allocation** as `maap-dps-worker-32gb`.
-      - Enter the **Container URL** as `mas.dit.maap-project.org/root/maap-workspaces/base_images/r:dit`.
-   1. Under Input Types, add the following Name, Description pairs:
-      1. For File Inputs:
-         - `aoi`: Area of Interest GeoJSON file
-      1. For Positional Inputs:
-         - `doi`: Digital Object Identifier (DOI) of the GEDI collection to subset, or a logical name representing such a DOI
-         - `temporal`: (Default: full temporal range): Temporal range to subset.
-         - `lat`: Name of the dataset used for latitude.
-         - `lon`: Name of the dataset used for longitude.
-         - `beams`: (Default: `all`): `all`, `coverage`, or `power`, or a comma-separated list of specific beam names, with or without the `BEAM` prefix (e.g., `BEAM0000,BEAM0001` or `0000,0001`).
-         - `columns`: One or more column names, separated by commas, to include in the output file.
-         - `query`: (Default: no query): Query expression for subsetting the rows in the output file.
-         - `limit`: (Default: 1_000): Maximum number of GEDI granule data files to download from the CMR
-         - `output`: Name to use for the output file.
-1. Once finished, click **Register Algorithm**
-1. Check the build job status at
-   <https://repo.maap-project.org/root/register-job-hysds-v4/-/jobs>.  If the job
-   fails, you will need to correct the issue (and likely create a patch release,
-   following the release steps again).  Otherwise, you should now be able to
-   open the **Jobs** menu, choose **View & Submit Jobs**, and find
-   the new version of the algorithm in the dropdown list of the **Submit** tab.
-
-## Citations
-
-Country Boundaries from:
-
-Runfola, D. et al. (2020) geoBoundaries: A global database of political
-administrative boundaries.  PLoS ONE 15(4): e0231866.
-<https://doi.org/10.1371/journal.pone.0231866>
-
-[conda installation]:
-   https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html
-[Digital Object Identifier]:
-   https://doi.org
-[geoBoundaries]:
-  https://www.geoboundaries.org
-[geoBoundaries API]:
-  https://www.geoboundaries.org/api.html
-[Keep a Changelog]:
-  https://keepachangelog.com/en/1.0.0/
-[MAAP GEDI Subsetter hosted on GitHub]:
-  https://github.com/MAAP-Project/gedi-subsetter.git
-[MAAP GitLab]:
-  https://repo.maap-project.org/
-[NASA MAAP]:
-  https://maap-project.org/
+## Beyond
+The Subsetter could be generalized to work with other datasets, pass back different data formats, directly read from S3, and more. The Subsetter is a great example of how MAAP can help users with their data processing needs.
```

Comment:
* (trivial) blank line after heading
* perhaps also mention that a future item would be to provide an example of usage outside of the MAAP